src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <section class="productCard">
+      <section class="productCard__card card" data-qa="card">
+        <div class="card__image image">
+        </div>
+        <h3 class="card__title title">
+          APPLE A1419 iMac 27" Retina 5k Monoblock (MNED2UA/A)
+        </h3>
+        <div class="card__code code">Product code: 195434</div>
```

Comment:
in h3 you put the content on a new line, in div you don't. why? be consistent

also do not use divs for text. divs are used to group other elements. for text use the `p` tag

Reply:
thanks 
in new version i used <p> tag